### ISSUE TITLE
Selección de usuario 

### ISSUE THREAD

**Issue by Jehydi** [LMS-Efpem/LMS#2]
¿Podríamos eliminar dropbox del los usuarios y solo dejar  usuario y contraseña?

**Comment by mbrend** [comment]
Sí, estoy de acuerdo. Eliminemos y dejemos solo el campo de usuario y contraseña.
